lib/install/config/loaders/installers/elm.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  test: /\.elm$/,
+  exclude: [/elm-stuff/, /node_modules/],
+  loader: 'elm-hot-loader!elm-webpack-loader?verbose=true&warn=true&debug=true'
```

Comment:
Hi, I'm the author of the webpack loader. It's _highly_ recommended that you set the [cwd](https://github.com/elm-community/elm-webpack-loader#cwd-default-null-recommended) option